engine/collection/compliance/engine.go

```diff
@@ -109,7 +109,7 @@ func NewEngine(
 			Match: func(msg *engine.Message) bool {
 				_, ok := msg.Payload.(*messages.ClusterBlockProposal)
 				if ok {
-					core.metrics.MessageReceived(metrics.EngineClusterCompliance, metrics.MessageClusterBlockProposal)
+					core.metrics.MessageReceived(metrics.EngineClusterCompliance, metrics.MessageClusterBlockProposal, msg.OriginID)
```

Comment:
after digging a bit, adding the nodeID as a label will significantly increase the number of time series that prometheus would need to track. for now, let's stick with just including the nodeIDs in the logs and extract metrics from there.

Reply:
https://github.com/onflow/flow-go/pull/3270/commits/fad2b8f9bf81df6a7ca3fa9f33b277a6782a7182